### PR TITLE
New messages

### PR DESCRIPTION
**Pull request by zorkian**

### CODE REVIEW

**Comment by zorkian** [comment]
cc @DrTall maybe

File: broker.go [hunk 1]
```diff
@@ -390,24 +390,35 @@ func (b *Broker) coordinatorConnection(consumerGroup string) (conn *connection,
 		}
 
 		if conn != nil {
-			defer func(lconn *connection) { go b.conns.Idle(lconn) }(conn)
-			resp, err := conn.ConsumerMetadata(&proto.ConsumerMetadataReq{
-				ClientID:      b.conf.ClientID,
-				ConsumerGroup: consumerGroup,
-			})
+			// Run this in a closure so we can ensure we release the connection
```
Comment (DrTall) [inline broker.go]: Probably easier to read as a real function considering that this closure interacts with `conn` that then gets overwritten on line 423 but I don't feel strongly.

**Review by DrTall — APPROVED** [review]
LGTM

**Review by DrTall — APPROVED** [review]
LGTM

**Pull request merged**